test/e2e/ModuleTest_Template.txt

```diff
@@ -1,61 +0,0 @@
-// SPDX-License-Identifier: LGPL-3.0-only
```

Comment:
What was this file used for? Just curious

Reply:
Yeah my bad
I think initially this was used as an copy paste example of how to structure e2e tests,
but in my cleanup rage I deleted it. 😅
I reverted the change, but I would argue, that with our repo still constantly changing this might be something that we might not think of when we make changes to the e2e tests in general
@0xNuggan any opinion on this, because you created it?